homeassistant/components/awair/strings.json

```diff
@@ -48,5 +48,11 @@
       "unreachable": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "flow_title": "{model} ({device_id})"
+  },
+  "issues": {
+    "prefer_local": {
+      "title": "Awair Local integration available",
+      "description": "Awair is currently configured to use the Cloud API. This is only recommended when the device is not connected to the same network as Home Assistant, or if you have a legacy device. Consider enabling the Local API on your device, deleting this integration, and reconfiguring the integration locally."
```

Comment:
This is advice, but not something one should repair.
Repairs are for things that should be actioned, otherwise causing issues now or in the known near future.

Reply:
Relying on the cloud integration can cause issues, e.g. #64497, so there is a strong desire to push people towards the local API. I wonder if repairs should be extended to general recommendations like this.

Reply:
> I wonder if repairs should be extended to general recommendations like this.

I think logs and docs are better places for that. If we push everything into repairs, it will result in people being annoyed. We currently reserve them for things that need fixing.

PS: #64497 is about hitting rate limits, which should be better controlled by the integration in that case. We cannot fix that by saying, "Use local", we need to fix that by preventing we hit those limits.